T16066: exit never happens
CallStack (from HasCallStack):
  error, called at T16066.hs:27:3 in main:Main
HasCallStack backtrace:
CallStack (from HasCallStack):
  collectBacktraces, called at libraries/ghc-internal/src/GHC/Exception.hs:88:13 in ghc-internal:GHC.Exception
  toExceptionWithBacktrace, called at libraries/ghc-internal/src/GHC/Exception.hs:124:3 in ghc-internal:GHC.Exception



